pandas/io/tests/parser/common.py

```diff
@@ -45,6 +46,21 @@ def test_empty_decimal_marker(self):
         with tm.assertRaisesRegexp(ValueError, msg):
             self.read_csv(StringIO(data), decimal='')
 
+    def test_bad_stream_exception(self):
+        handle = open(self.csv_shiftjs, "rb")
```

Comment:
add the issue number as a comment (and a 1-liner explaining what this test is catching)